### ISSUE TITLE
Behaviour of DISABLE_ACCESS_LOG is misleading

### ISSUE THREAD

**Issue by pakal** [jazzband/django-axes#446]
The docs says this :

```
AXES_DISABLE_ACCESS_LOG:  If True,  disable  all  access  logging,  so  the  admin interface will be empty. Default:False
```

However, at the current state of the code, AXES_DISABLE_ACCESS_LOG is only used in user_logged_out() function, to prevent updating logout times on success logs, and unsuccessful login attempts seem to never be logged into "Access logs" table. 
So I don't get what this setting shoudl actually control.

**Comment by aleksihakli** [comment]
Hey @pakal! Good catch, thanks for reporting the issue.

The `axes.models.AccessAttempt` objects are used to monitor the user login failures.

The `axes.models.AccessLog` objects are used to track successful user logins and logouts in the [database handler](https://github.com/jazzband/django-axes/blob/41c4eda0471d4d6a4f0ef89164d57c7fa046b13a/axes/handlers/database.py) `user_logged_in` and `user_logged_out` functions which are called by the Django signal handlers.

The `AXES_DISABLE_ACCESS_LOG` is currently used to toggle only the user logout logging to the database in  `axes.handlers.AxesDatabaseHandler.user_logged_out`. The success seems to be controlled with the `AXES_DISABLE_SUCCESS_ACCESS_LOG` flag. The functionality should indeed be updated so that the success logging is also disabled when `AXES_DISABLE_ACCESS_LOG = True`:

https://github.com/jazzband/django-axes/blob/41c4eda0471d4d6a4f0ef89164d57c7fa046b13a/axes/handlers/database.py#L176-L183

It seems like there is redundancy in the code and the `AXES_DISABLE_ACCESS_LOG` flag should be used in the success logging function in `axes.handlers.AxesDatabaseHandler.user_logged_in` instead of the current `AXES_DISABLE_SUCCESS_ACCESS_LOG` flag:

https://github.com/jazzband/django-axes/blob/41c4eda0471d4d6a4f0ef89164d57c7fa046b13a/axes/handlers/database.py#L147-L156

It seems like the condition should really be:

```python
if not settings.AXES_DISABLE_ACCESS_LOG:
    AccessLog.objects.create(...)
```

The `AXES_DISABLE_SUCCESS_ACCESS_LOG` should most probably be removed, since there are two redundant flags that handle the same objects. The current situation with two overlapping flags probably stems from the historical features that were implemented before version 5, where the database and cache logic was separated into different handler classes, and the code was a lot more entangled. After refactoring the logic was left as it is so as to not break backwards compatibility.

The documentation should probably read that setting the `AXES_DISABLE_ACCESS_LOG` flag disables the creation of `AccessLog` records when the database handler is being used, and `AccessAttempt` records are still created. The database handlers relies on using the database for access monitoring, and disabling the use of `AccessAttempts` would make absolutely no sense.

If we remove the `AXES_DISABLE_SUCCESS_ACCESS_LOG`, we should add a Django check which warns if the user has configured that setting somewhere in their project and advice them to deprecate that flag, and use the `AXES_DISABLE_ACCESS_LOG` flag instead.

**Comment by aleksihakli** [comment]
Hey @pakal, I fixed the documentation and released a new version 5.0.6 with better flag semantics.

**Comment by pakal** [comment]
I looked at the patch, it seems to resolve ambiguities perfectly   :)
Thanks !
